composer.json

```diff
@@ -9,17 +9,19 @@
             "email": "dev-team@ez.no"
         }
     ],
+    "minimum-stability": "dev",
+    "prefer-stable": true,
     "require": {
-        "php": ">=7.1",
+        "php": "^7.3",
         "ezsystems/ezpublish-kernel": "^8.0@dev",
-        "ocramius/proxy-manager": "~1.0 || ~2.0",
-        "symfony/proxy-manager-bridge": "^2.8.40 || ^3.4.11",
-        "zetacomponents/system-information": "^1.1.1",
-        "ezsystems/ezplatform-admin-ui": "^2.0@dev"
+        "ezsystems/ezplatform-admin-ui": "^2.0@dev",
+        "ocramius/proxy-manager": "^2.2",
+        "symfony/proxy-manager-bridge": "^4.3",
+        "zetacomponents/system-information": "^1.1.1"
     },
     "require-dev": {
         "friendsofphp/php-cs-fixer": "~2.14.2",
-        "phpunit/phpunit": "^5.7.27 || ^6.5.14"
+        "phpunit/phpunit": "^8.2"
```

Comment:
Is it correct? Other repositories specifies `phpunit/phpunit` dependency as `^8.1` 

* https://github.com/ezsystems/ezplatform-admin-ui/pull/966/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R51
* https://github.com/ezsystems/ezplatform-user/pull/37/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R42

Reply:
Yes. In the meantime PHPUnit 8.2 has been released. I will update other packages to install 8.2.